ebaysdk/inventorymanagement/__init__.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+
+'''
+Authored by: Michal Hernas
+Licensed under CDDL 1.0
+'''
+
+import os
+
+from ebaysdk import log
+from ebaysdk.connection import BaseConnection
+from ebaysdk.exception import RequestPaginationError, PaginationLimit
+from ebaysdk.config import Config
+from ebaysdk.utils import dict2xml
+
+class Connection(BaseConnection):
+    """Connection class for the Inventory Management service
+
+    API documentation:
+    http://developer.ebay.com/Devzone/store-pickup/InventoryManagement/index.html
+
+    Supported calls:
+    AddInventory
+    AddInventoryLocation
+    DeleteInventory
+    DeleteInventoryLocation
+    (all others, see API docs)
+
+    Doctests:
+    Create location first
+    >>> f = Connection(config_file=os.environ.get('EBAY_YAML'), debug=False)
+    >>> retval = f.execute('AddInventoryLocation', {
+    ...     'Address1': u'Alexanderplatz 12',
+    ...     'Address2': u'Gebaude 6',
+    ...     'City': u'Berlin',
+    ...     'Country': u'DE',
+    ...     'PostalCode': u'13355',
+    ...     'Latitude': u'37.374488',
+    ...     'Longitude': u'-122.032876',
+    ...     'LocationID': u'ebaysdk_test',
+    ...     'LocationType': u'STORE',
+    ...     'Phone': u'(408)408-4080',
+    ...     'URL': u'http://store.com',
+    ...     'UTCOffset': u'+02:00',
+    ...     'Name': 'Test',
+    ...     'Region': 'Berlin',
+    ...     'PickupInstructions': 'Pick it up soon',
+    ...     'Hours': [{'Day': {'DayOfWeek': 1, 'Interval': {'Open': '08:00:00', 'Close': '10:00:00'}}}]
+    ...     })
+    >>> error = f.error()
+    >>> if not f.error():
+    ...   print(f.response.reply.LocationID.lower())
+    ebaysdk_test
+
+    And now add item it it
+    >>> f = Connection(config_file=os.environ.get('EBAY_YAML'), debug=False)
+    >>> retval = f.execute('AddInventory', {"SKU": "SKU_TEST", "Locations": {"Location": [
+    ...    {"Availability": "IN_STOCK", "LocationID": "ebaysdk_test", "Quantity": 10}
+    ...     ]}})
+    >>> error = f.error()
+    >>> if not f.error():
+    ...   print(f.response.reply.SKU.lower())
+    sku_test
+
+
+    Delete item from all locations
+    >>> f = Connection(config_file=os.environ.get('EBAY_YAML'), debug=False)
+    >>> retval = f.execute('DeleteInventory', {"SKU": "SKU_TEST", "Confirm": 'true'})
+    >>> error = f.error()
+    >>> if not f.error():
+    ...   print(f.response.reply.SKU.lower())
+    sku_test
+
+
+    Delete location
+    >>> f = Connection(config_file=os.environ.get('EBAY_YAML'), debug=False)
+    >>> retval = f.execute('DeleteInventoryLocation', {"LocationID": "ebaysdk_test"})
+    >>> error = f.error()
+    >>> if not f.error():
+    ...   print(f.response.reply.LocationID.lower())
+    ebaysdk_test
+
+    """
+
+    def __init__(self, **kwargs):
+        """Finding class constructor.
+
+        Keyword arguments:
```

Comment:
please make sure these defaults match the actual default. For example, default domain should be api.ebay.com